test/runner.py

```diff
@@ -564,6 +564,8 @@ def set_env(name, option_value):
 
   # Remove any old test files before starting the run
   utils.delete_file(common.flaky_tests_log_filename)
+  utils.delete_file(common.browser_spawn_lock_filename)
+  utils.delete_file(f'{common.browser_spawn_lock_filename}_counter')
```

Comment:
Is it not possible to store the count in the lock file itself?  (i.e. can we avoid separate counter file?)

Reply:
I did that first, but got odd behavior with O_EXCL when O_CREAT was not always unconditionally passed. So reverted to this simpler two-file form